common/djangoapps/entitlements/helpers.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime, timedelta
+
+import pytz
+import sys
+from django.conf import settings
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+def is_entitlement_expired(entitlement):
+    """
+    Determines from the policy if an entitlement can be redeemed, if it has not passed the
+    expiration period of policy.expiration_period_days, and has not already been redeemed
+    """
+    utc = pytz.UTC
+    days_since_created = (datetime.utcnow().replace(tzinfo=utc) - entitlement.created).days
+    expiration_period = settings.ENTITLEMENTS_POLICY.get('expiration_period_days', sys.maxint)
+    if entitlement.policy:
+        expiration_period = entitlement.policy.expiration_period_days
+    return days_since_created > expiration_period and not entitlement.enrollment_course_run
+
+
+def is_entitlement_refundable(entitlement):
+    """
+    Determines from the policy if an entitlement can still be refunded, if the entitlement has not
+    yet been redeemed (enrollment_course_run is NULL) and policy.refund_period_days has not yet passed
+    """
+    utc = pytz.UTC
+    days_since_created = (datetime.utcnow().replace(tzinfo=utc) - entitlement.created).days
```

Comment:
This is repeated in https://github.com/edx/edx-platform/pull/16627/files#diff-67dc6b36367a82550b5213e2e9af5d28R16, maybe it could be extracted into its own helper method?

Reply:
:+1: